recipe/meta.yaml

```diff
@@ -23,7 +23,8 @@ requirements:
     - pillow
     - pretrainedmodels ==0.7.4
     - python >=3.6
```

Comment:
```suggestion
    - python >=3.7
```

Upstream says that this should be 3.7.
https://github.com/qubvel/segmentation_models.pytorch/blob/v0.3.3/setup.py#L20

though it may be entirely true that 3.6 is compatible.

Reply:
Minimum version for python is 3.9 now according to https://github.com/qubvel-org/segmentation_models.pytorch/blame/v0.3.4/setup.py#L20